eng/tools/azure-sdk-tools/ci_tools/logging/__init__.py

```diff
@@ -4,9 +4,40 @@
 import os
 import datetime
 from subprocess import run
+import argparse
 
 LOGLEVEL = getattr(logging, os.environ.get("LOGLEVEL", "INFO").upper())
 
+logger = logging.getLogger("azure-sdk-tools")
+
+def configure_logging(
+    args: argparse.Namespace,
+    level: str = "INFO",
+    fmt: str = "%(asctime)s [%(levelname)s] %(name)s: %(message)s"
+) -> None:
+    """
+    Configures the shared logger. Should be called **once** at startup.
+    """
+    
+    numeric_level = getattr(logging, level.upper(), None)
+
+    # parse cli arg
+    if args.quiet:
```

Comment:
I think we should disallow the presence of --log-level if the user provides --quiet or --verbose arguments. It needs to be one or the other.